docsets/openFrameworks/docset.json

```diff
@@ -9,6 +9,10 @@
     "aliases": ["of"],
 
     "specific_versions": [
+        {
+			"version": "0.11.0",
+			"archive": "versions/0.11.0/openFrameworks.tgz"
```

Comment:
You need to add the archive at this path as well. Did you add it to `versions/0.10.1/openFrameworks.tgz` by mistake?

Reply:
sorry I forgot that.
just put new archive in 0.11.0